collect_app/src/main/java/org/odk/collect/android/activities/FormHierarchyActivity.java

```diff
@@ -565,7 +562,7 @@ private void refreshView(boolean isGoingUp) {
                 // retrieve the current group
                 TreeReference curGroup = (visibleGroupRef == null) ? contextGroupRef : visibleGroupRef;
 
-                if (!curGroup.isParentOf(currentRef, false)) {
+                if (curGroup != null && !curGroup.isParentOf(currentRef, false)) {
```

Comment:
Explanation: Check for `null` here because now `contextGroupRef` may not have been set.